pkg/retriever/executor_test.go

```diff
@@ -104,21 +104,26 @@ func TestQueryFiltering(t *testing.T) {
 			for p := range tc.queryResponses {
 				candidates = append(candidates, types.RetrievalCandidate{MinerPeer: peer.AddrInfo{ID: peer.ID(p)}})
 			}
-			mockCandidateFinder := &testutil.MockCandidateFinder{Candidates: map[cid.Cid][]types.RetrievalCandidate{cid.Undef: candidates}}
-			retriever := &Retriever{config: RetrieverConfig{PaidRetrievals: tc.paid}} // used for isAcceptableQueryResponse() only
 
-			cfg := &RetrievalConfig{
+			cfg := &Executor{
 				GetStorageProviderTimeout:   func(peer peer.ID) time.Duration { return time.Second },
 				IsAcceptableStorageProvider: func(peer peer.ID) bool { return true },
-				IsAcceptableQueryResponse:   retriever.isAcceptableQueryResponse,
```

Comment:
To be really honest, the main reason this line is even in here is to get coverage %'s up because otherwise that function wasn't covered. So it's not really essential to duplicate the functionality, it's just important to have it do something interesting.
I don't think the tests in here really fully exercise the full function; but this is fine.